lib/web/appserver.rb

```diff
@@ -114,6 +114,17 @@ def value_to_msg(value)
   def notepad_text_path
     File.join(Narou.local_setting_dir, "notepad.txt")
   end
+
+  def query_to_boolean(value, default: false)
+    case value
+    when "1", 1, "true", true
+      true
+    when  "0", 0, "false", false
+      false
+    else
+      default
+    end
```

Comment:
__[rubocop]__
* Metrics/ClassLength: Class has too many lines. [697/300]
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->